### ISSUE TITLE
Changing target language

### ISSUE THREAD

**Issue by barn852** [EdgeTranslate/EdgeTranslate#556]
One of the best translate extension, I just love it.

Most of the time I use it for translating English to my mother tongue (Hungarian) and vice versa (!) - mutual translation mode is ON.
However when I want to translate from other languages different from English, I have to change the translation preferences in the settings.

If I turn on the Auto Detect for the source language, I cannot use the mutual mode (apparently).

I would like to suggest a roll down menu for quick language change added to the result frame.

![image](https://github.com/EdgeTranslate/EdgeTranslate/assets/68149741/f890846e-d215-42f2-9f11-cafc4c16bb2d)

**Comment by YunFeng0817** [comment]
Good suggestion!

**Comment by DvoeglazovIlya** [comment]
Yes, it is a problem. I change language from English to Russian here:
![image](https://github.com/EdgeTranslate/EdgeTranslate/assets/15049055/b6f2835e-6ce7-4c2b-a21c-5041b40c8b89)
